jupyterlab_comments/jupyterlab_comments/git_commands.py

```diff
@@ -63,16 +63,35 @@ def get_comments_for_path(self, file_path_from_repo_root, git_root_dir):
 		Returns a JSON list of commments for the given file.
 
 		Assumes that git_root_dir is a valid path within a Git repo.
-		Keys of objects returned: hash, comment, original, **children
+		Keys of objects returned: hash, comment, original, children
 		"""
 		#self.appraise_pull(current_path) #pull new comments from remote repo
 		comments_string = self.run(git_root_dir, 'appraise', 'show', '-d',
 									'-json', file_path_from_repo_root)
 		comments_json = json.loads(comments_string)
 		return comments_json
 
-	def get_code_review_comments(self, file_path, server_root):
-		pass
+	def get_code_review_comments(self, file_path_from_repo_root, git_root_dir):
+		"""
+		Returns the JSON for the current code review with comments for the requested file path.
+
+		Returns None of there are no review comments for the file path.
+		"""
+		#self.appraise_pull(current_path) #pull new comments from remote repo
+		review_string = self.run(git_root_dir, 'appraise', 'show',
+									'-json')
+		review_json = json.loads(review_string)
+		if "comments" in review_json.keys():
```

Comment:
This condition doesn't seem right.

If there is a review with no comments, then you will return `None`, but if there is a review with no comments on this file, then you will return a review with an empty comments field.

I would suggest re-structuring this code somewhat:

1. Change the `else` clause to be a stand-alone condition that short-circuits the rest of the method;

```py
        if not review_json:
            return None
```

2. Then, reduce the indentation of the rest of the code.
3. Instead of explicitly checking if `comments` is in the dictionary, fetch the comments using `review_json.get("comments", [])`. That way the code is slightly simpler and you also treat the conditions of no comments vs no-matching-comments identically.